highlighters/main/main-highlighter.zsh

```diff
@@ -178,7 +180,9 @@ _zsh_highlight_main__type() {
     if (( $+aliases[(e)$1] )); then
       may_cache=0
     fi
-    if (( $+aliases[(e)$1] )) && (( aliases_allowed )); then
+    if false && (( ${+galiases[(e)$1]} )); then
```

Comment:
TODO: remove the `false &&` before merging; I just put it here to test the fallback codepath.